noxfile.py

```diff
@@ -7,15 +7,15 @@
 
 nox.options.reuse_existing_virtualenvs = True
 
-PYTHON_VERSIONS = ['pypy3', '3.6', '3.7', '3.8', '3.9']
+PYTHON_VERSIONS = ['pypy3', '3.7', '3.8', '3.10']
```

Comment:
why was 3.9 removed? and is 3.6 no longer supported? if so, explain why in your pr body

Reply:
it is a mistake for python3.9, sorry
and yes python3.6 is will be no more supported, I will add a comment